app/views/editorial/submissions/node_types/_news_article.html.haml

```diff
@@ -0,0 +1,3 @@
+
+%dt Release date
+%dd= content["date"]
```

Comment:
I dont think this is right - it should be `release_date`, and it's also not revisable so I dont think it would be in revision_content? I think we need to get `release_date` from the NewsArticle node.